src/lib/components/ContentContainer.svelte

```diff
@@ -1,47 +1,9 @@
-<script lang="ts">
-    import type { HTMLImgAttributes } from 'svelte/elements';
-
-    // eslint-disable-next-line init-declarations
-    export let topText: string;
-    // eslint-disable-next-line init-declarations
-    export let headerText: string;
-    // eslint-disable-next-line init-declarations
-    export let bottomText: string;
-
-    // eslint-disable-next-line init-declarations
-    export let imageText1: string;
-    // eslint-disable-next-line init-declarations
-    export let imageText2: string;
-
-    // eslint-disable-next-line init-declarations
-    export let src1: HTMLImgAttributes['src'];
-    // eslint-disable-next-line init-declarations
-    export let src2: HTMLImgAttributes['src'];
-
-    // eslint-disable-next-line init-declarations
-    export let alt1: HTMLImgAttributes['alt'];
-    // eslint-disable-next-line init-declarations
-    export let alt2: HTMLImgAttributes['alt'];
-</script>
-
 <div
-    class="flex flex-col items-center justify-between gap-12 rounded-3xl bg-warm-white px-8 py-8 md:px-32 md:py-16 lg:flex-row"
+    class="my-6 flex flex-col items-center justify-between gap-12 rounded-3xl bg-warm-white px-8 py-8 md:px-32 md:py-16 lg:flex-row"
```

Comment:
We typically avoid introducing margins at the component-level. Instead, we delegate that responsibility to the caller (i.e., the parent component) for the greatest flexibility.
```suggestion
    class="flex flex-col items-center justify-between gap-12 rounded-3xl bg-warm-white px-8 py-8 md:px-32 md:py-16 lg:flex-row"
```